api/src/main/java/org/openmrs/module/kenyaemr/reporting/data/converter/definition/evaluator/hei/HEIAgeAtTestInConfirmatoryDataEvaluator.java

```diff
@@ -28,12 +28,12 @@ public EvaluatedPersonData evaluate(PersonDataDefinition definition, EvaluationC
 
         String qry = "select\n" +
                 "  d.patient_id,\n" +
-                "  round(DATEDIFF(f.visit_date,d.DOB)/30) as age_at_test_twelve_months\n" +
```

Comment:
when calculating age. use DIV in place of "/" as this will eliminate need to round off.